src/NodeTraverser.php

```diff
@@ -69,7 +69,9 @@ protected function traverseForVisitor(NodeVisitorInterface $visitor, \Twig_NodeI
         $node = $visitor->enterNode($node, $this->env);
 
         foreach ($node as $k => $n) {
-            if (false !== ($m = $this->traverseForVisitor($visitor, $n)) && null !== $m) {
+            if (null === $n) {
+                $node->setNode($k, $n);
```

Comment:
this could be `continue` instead. You don't need to set it again (as it is already set) and it will avoid a deprecation here about setting `null`

Reply:
Using `continue` makes the code more verbose. In 2.x, this code is not needed as `$n` can never be `null`.

Reply:
why is `continue` more verbose than `$node->setNode($k, $n);` ?

Reply:
Oh, got it now :)